send_to_telemetry.py

```diff
@@ -81,7 +81,7 @@ def main(path):
             validate(asdict(result), schema)
 
         # send to telemetry
-        wpt_run_uuid = uuid.uuid4().hex
+        wpt_run_uuid = str(uuid.uuid4())
```

Comment:
`str(uuid)` returns the 36-character representation.

See https://docs.python.org/3.8/library/uuid.html#uuid.UUID

Reply:
Thanks, Jeff!